### DISCUSSION TITLE
Unable to inspect drift database

### DISCUSSION THREAD

**Discussion by BojanDolic** [simolus3/drift#1759] [Q&A]
I started using drift today and I built a simple table that contains id as primary key and name column which is text.
I tried to insert some dummy data and read it from the database and that works as expected.

Problem is, inside the database inspector inside the android studio, this database is not shown and I can't inspect tables and see column values at all.

**Comment by simolus3** [comment]
Are you using the new [Local Storage Inspector](https://plugins.jetbrains.com/plugin/18231-local-storage-inspector) IDEA plugin with the [`drift_local_storage_inspector`](https://pub.dev/packages/drift_local_storage_inspector) package? Those plugins replace the older drift inspector as far as I know.
